src/style-sheets/components/_toolbar-logo.scss

```diff
@@ -0,0 +1,18 @@
+.desktop-logo {
+    width: 148px;
+}
+
+.mobile-logo {
+    display: none;
+    width: 30px;
+}
+
+@include medium('down') {
```

Comment:
I think the media query needs to be flipped around.

Reply:
fixed